Source/ASTableView.h

```diff
@@ -37,6 +37,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface ASTableView (UIGestureRecognizerDelegate) <UIGestureRecognizerDelegate>
```

Comment:
Any reason to put this in a category instead of just having conformance above?

Reply:
I think using category would make the structure of `UIGestureRecognizerDelegate` implementations more explicit and distinct as an extension from class `ASTableView `. It is similar to the Swift syntax [Adding Protocol Conformance with an Extension](https://docs.swift.org/swift-book/LanguageGuide/Protocols.html#ID277) 🤔